qiskit/pulse/interfaces.py

```diff
@@ -11,127 +11,26 @@
 # that they have been altered from the originals.
 
 """
-ScheduleComponent, a common interface for components of schedule (Instruction and Schedule).
+NamedValue, a common interface for components of schedule (Instruction and Schedule) and beyond.
 """
 from abc import ABCMeta, abstractmethod
-from typing import Tuple, List, Union, Optional
-
+from typing import List
 from qiskit.pulse.channels import Channel
 
 # pylint: disable=missing-type-doc
 
 
-class ScheduleComponent(metaclass=ABCMeta):
-    """Common interface for components of schedule. """
+class NamedValue(metaclass=ABCMeta):
+    """Common interface for components of schedule and future interconnected classes. """
 
     @property
     @abstractmethod
     def name(self) -> str:
-        """Name of ScheduleComponent."""
+        """Name of NamedValue."""
         pass
 
     @property
     @abstractmethod
     def channels(self) -> List[Channel]:
```

Comment:
I don't think channels should be included here

Reply:
Removed

Reply:
Removed